site-scrapers/Atrius.js

```diff
@@ -0,0 +1,126 @@
+    console.log("Atrius done.");
+    return {
+        ...sites.Atrius,
+        ...webData,
+    };
+};
+
+async function ScrapeWebsiteData() {
+
+    // const page = await browser.newPage();
+    // await page.goto(sites.Atrius.website);
+
+    // We need to go through the flow and use a request verification token
+    let cookie = ""
+    const initialPromise = new Promise((resolve) => {
+        let response = "";
+        https.get(sites.Atrius.website, (res) => {
+            let body = "";
+            res.on("data", (chunk) => {
+                body += chunk;
+            });
+            res.on("end", () => {
+                response = body;
+
+                resolve(response);
+            });
+            responseHeaders = res.headers
+            cookie = responseHeaders['set-cookie'];
+        });
+    });
+    const initialResponse = await initialPromise;
+    console.log('cookie: ' + cookie);
+
+    
+    const dphPromise = new Promise((resolve) => {
+        https.get(sites.Atrius.dphLink, (res) => {
+            let body = "";
+            responseHeaders = res.headers
+            // update the cookie again
+            cookie = responseHeaders['set-cookie'];
+            res.on("data", (chunk) => {
+                body += chunk;
+            });
+            res.on("end", () => {
+                response = body;
+
+                resolve(response);
+            });
+        });
+    });
+    const dphResponse = await dphPromise;
+    const root = html_parser.parse(dphResponse)
+    verificationInput = root.querySelector('input[name=__RequestVerificationToken]')
+    verificationToken = verificationInput.getAttribute('value');
+
+    // Setup the post data to answer the survey questions
+    const postData = 'view=grouped&specList=121&vtList=1424&start=&filters=%7B%22Providers%22%3A%7B%7D%2C%22Departments%22%3A%7B%7D%2C%22DaysOfWeek%22%3A%7B%7D%2C%22TimesOfDay%22%3A%22both%22%7D'
+    const options = {
+        hostname: 'myhealth.atriushealth.org',
+        port: 443,
+        path: '/OpenScheduling/OpenScheduling/GetScheduleDays',
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
+          'Content-Length': postData.length,
+          '__RequestVerificationToken': verificationToken,
+          'Cookie': cookie
+        }
+      }
+
+    // now request the schedule
+    const schedulePromise = new Promise((resolve) => {
+        let response = "";
+        const req = https.request(options, (res) => {
+            let body = "";
+            responseHeaders = res.headers
+            res.on("data", (chunk) => {
+                body += chunk;
+            });
+            res.on("end", () => {
+                response = JSON.parse(body);
+                resolve(response);
+            });
+        });
+        req.write(postData);
+        req.end();
+    });
+
+    const postResponse = await schedulePromise;
+    allDays = postResponse.AllDays
```

Comment:
missing semicolon; I suggest installing "prettier" and using the default (I use 4 spaces for a tab - I'll put a config file in for this repo to make things more consistent)

Reply:
Did you have a `prettier` config you wanted to use? I used the below. I'm happy to add that if it works.
```yml
# .prettierrc or .prettierrc.yaml
trailingComma: "es5"
tabWidth: 4
semi: true
singleQuote: true
```

Reply:
DMed you on slack - that looks good except I think I've been using " throughout the repo so we'd want singleQuote false IIRC